device/rsvp/rsvpEro.yaml

```diff
@@ -0,0 +1,74 @@
+components:
+  schemas:
+    Rsvp.Ero:
+      description: >-
+        Configuration for the optional RSVP-TE ERO object included in Path Messages. 
+      type: object
+      properties:
+        prepend_neighbor_ip:
+          description: >-
+            Determines whether the IP address of the RSVP neighbor should be added as an ERO sub-object.
+            If it is to be included, it can be included as a Loose hop or as a Strict hop.
+          type: string
+          enum:
+          - dont_prepend
+          - prepend_loose
+          - prepend_strict
+          default: prepend_loose
+        prefix_length:
+          description: >-
+            If prepend_egress_ip is set to one of 'prepend_loose' or 'prepend_strict', then set this value as the prefix length of the ERO sub-object containing egress IP address.                        
+          type: integer
+          default: 32
+          minimum: 0
+          maximum: 32
```

Comment:
will this object ever be used by IPv6?  If yes, what would be a reasonable max here?

Reply:
If Rsvp V6 is needed (we don't foresee Rsvp v6 requirement in mid to long term) we can change the max to 128 without breaking backward compatibility.

Reply:
Yes, the plan is to add an extra enum for IPv6 . We have also modified ipv4_prefix_length to prefix_length so that it can be re-used for IPv6 sub-object if and when it comes in.